data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/serializers/SimpleStateSerializer.java

```diff
@@ -24,16 +24,16 @@
 /**
  * @author Alexandre Montigny {@literal <alexandre.montigny at rte-france.com>}
  */
-public class SimpleStateSerializer extends JsonSerializer<SimpleState> {
+public class SimpleStateSerializer extends JsonSerializer<PostContingencyState> {
```

Comment:
nename it PostContingencyStateSerializer ?

Reply:
Actually states are not serialized anymore, so I can just delete it...